### PR TITLE
Fix subject-filter for personal overview.

### LINKED ISSUE #5388: Schlagwort-Filter funktioniert auf persönlicher Übersicht nicht mehr.
Es werden keine Schlagwörter mehr vorgeschlagen, obwohl Dossiers Schlagwörter haben.

screenshot 2019-02-14 at 19 18 34

Siehe https://basecamp.com/2768704/projects/12554551/todos/380048323

### PR DESCRIPTION
**Pull request by elioschmutz**
Fixes #5388 

screen2

### CODE REVIEW

File: opengever/tabbedview/browser/base_tabs.py [hunk 1]
```diff
@@ -163,9 +163,11 @@ def _additional_solr_subject_filters(self):
         if hasattr(self, 'object_provides') and self.object_provides:
             filters.append('object_provides:{}'.format(self.object_provides))
 
-        if hasattr(self, 'search_options'):
-            for option, value in self.search_options.items():
-                filters.append('{}:{}'.format(option, str(value)))
+        # `search_options` can contain functions as values. Using
+        # `_search_options` instead `search_options` ensures that callables
```
Comment (njohner) [inline opengever/tabbedview/browser/base_tabs.py]: instead of

File: opengever/tabbedview/browser/base_tabs.py [hunk 2]
```diff
@@ -163,9 +163,11 @@ def _additional_solr_subject_filters(self):
...
+        # are called properly and we have the real value of the option.
+        for option, value in self._search_options.items():
```
Comment (njohner) [inline opengever/tabbedview/browser/base_tabs.py]: As `GeverTabMixin` does not have 'search_options' as attribute, I would rather keep the test that the attribute exists here (`hasattr(self, 'search_options')` or even `hasattr(self, '_search_options')`). Even if this is protected by `subject_filter_available`...

Comment (elioschmutz) [reply opengever/tabbedview/browser/base_tabs.py]: 👍 updated

**Comment by elioschmutz** [comment]
@deiferni ...

**Pull request merged**